### PR TITLE
Initialize

### PR DESCRIPTION
**Pull request by nbokulich**

### CODE REVIEW

File: q2_phylogenomics/__init__.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2017-2019, QIIME 2 development team.
```
Comment (thermokarst) [inline q2_phylogenomics/__init__.py]:
```suggestion
# Copyright (c) 2020, QIIME 2 development team.
```

Comment (thermokarst) [reply q2_phylogenomics/__init__.py]: Don't forget to make that change in all the other copyright headers (and LICENSE).

Comment (nbokulich) [reply q2_phylogenomics/__init__.py]: done, thanks!

Comment (thermokarst) [reply q2_phylogenomics/__init__.py]: I hate to be "that guy" - but the [LICENSE](https://github.com/qiime2/q2-phylogenomics/blob/a1ab15a04570257fbf161dcb303ff5eef39ba37d/LICENSE#L3) doesn't match, it should read

```
Copyright (c) 2020, QIIME 2 development team.
```

A good way to double-check all of this is to run `q2lint` locally.

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    author="Nicholas Bokulich",
+    author_email="nbokulich@gmail.com",
+    description="Phylogenomics toolkit and pipelines.",
+    url="https://qiime2.org/",
+    entry_points={
+        'qiime2.plugins':
+        ['q2-phylogenomics=q2_phylogenomics.plugin_setup:plugin']
+    },
+    package_data={
+        'q2_phylogenomics.tests': ['data/*']
```
Comment (ebolyen) [inline setup.py]: You'll want the bibtex file in here too

**Review by ebolyen — APPROVED** [review]

**Pull request merged**